tests/integration/base.py

```diff
@@ -2,6 +2,9 @@
 import os
 import random
 import requests
+from logging import getLogger
+
+logger = getLogger(__name__)
```

Comment:
I see you added this logger but is not used, did you add it to replace the `print(..)` bellow?

Reply:
Oh, you are correct. I used it during development but it got removed before the committed version. Good catch.